lib/cache-group.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var Cache = require('./cache');
+
+/*
+ * CacheGroup is used to both speed up, and ensure consistency of hashForDep.
+ *
+ * The CacheGroup contains 3 seperate caches:
+ *
+ *  - MODULE_ENTRY: the cache of PackageCacheEntry objects with information
```

Comment:
can we surface the  cache key, and cache value of MODULE_ENTRY cache. Similarly to what you did for `PATH` bellow

Reply:
Question for you: is there a reason that we're using a hash at all for these keys as opposed to just the actual string value? Just a shorter string? Something else? Let me take a look about what was done with PATH (I'm on the plane and having connection issues, so getting source files right now is an issue.)

Reply:
Stef, I lost the context of what you mean above - where is the 'PATH below' now that you're talking about? Also, why do you want to surface the cache key and value of MODULE_ENTRY? What sort of use-case are you thinking of?

Reply:
for `PATH` you have `nameDirKey => realPathKey.` which is really nice and descriptive, as to what the cached key/value pair are. 

Could you also surface that here for `MODULE_ENTRY`

Reply:
Oh, so you just mean changing the var names to be more descriptive? Sure, happy to fix that.

Reply:
Done. Will check in shortly.